spec/spec_helper.rb

```diff
@@ -27,7 +27,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = false
+  config.use_transactional_fixtures = true
```

Comment:
Ah ha, so this is some default Rails functionality which handles tests inside a transaction already, and we don't need DatabaseCleaner for most tests?

I read about it here: https://github.com/rails/rails/pull/19282
It has a new name which is clearer: `use_transactional_tests`. Can we use this now?

Also can we move the DatabaseCleaner section down below this (because it is now only for a couple of specific cases)?

I'd also suggest to add to the header comment to explain that this is what keeps the database clean (unfortunately there's no explanation of this in the [guide](https://guides.rubyonrails.org/testing.html) either)

Reply:
Turns out that Rails renamed it but Rspec kept the old name:

* https://github.com/rspec/rspec-rails/issues/1964#issuecomment-370975179